# Set speed
float64 speed